interactors/src/main/java/com/thefuntasty/interactors/disposables/FlowableDisposablesOwner.kt

```diff
@@ -0,0 +1,77 @@
+package com.thefuntasty.interactors.disposables
+
+import com.thefuntasty.interactors.interactors.BaseFlowabler
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.disposables.Disposable
+import io.reactivex.rxkotlin.plusAssign
+
+interface FlowableDisposablesOwner {
+
+    val disposables: CompositeDisposable
+
+    infix fun <ARGS, T> BaseFlowabler<ARGS, T>.executeWith(args: ARGS): Disposable {
+        return execute(args, { })
+    }
+
+    fun <ARGS, T> BaseFlowabler<ARGS, T>.execute(args: ARGS): Disposable {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/abd629572327cbabcfd5483dec148fd3bc6b5ac2/interactors/src/main/java/com/thefuntasty/interactors/disposables/FlowableDisposablesOwner.kt#L16"></span>Functions with exact one statement, the return statement, can be rewritten with ExpressionBodySyntax.</td>
    </tr>
  </tbody>
</table>